pages/blog/mistake-odyssey.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Mistake Odyssey"
+Subtitle: "Git, Github, VS Code and Markdown for non-programmers"
+date: 2020-07-08T06:00:00+01:00
```

Comment:
The year needs to be 2021 instead of 2020.

Been wondering where the blog went then saw it was among preview blogs :smile:

Reply:
Hahaha, yes, thanks :)